sopel/plugins/__init__.py

```diff
@@ -72,10 +75,14 @@ def find_internal_plugins():
 
 
 def find_sopel_modules_plugins():
-    """List plugins from ``sopel_modules.*``
+    """List plugins from ``sopel_modules.*``.
 
-    :return: Yield instance of :class:`~.handlers.PyModulePlugin`
+    :return: yield instance of :class:`~.handlers.PyModulePlugin`
              configured for ``sopel_modules.*``
+
+    Before entry point plugins, the only way to package a plugin was to follow
+    the :pep:`382` by using the ``sopel_modules`` namespace. This function is
```

Comment:
`:pep:` generates text of the form `PEP <number>`, so grammatically this sentence should read simply "follow PEP 382" when rendered.

```suggestion
    :pep:`382` by using the ``sopel_modules`` namespace. This function is
```